app/src/main/java/com/teamltt/carcare/activity/BaseActivity.java

```diff
@@ -0,0 +1,186 @@
+            NavigationView navigationViewBottom = (NavigationView) findViewById(R.id.nav_view_bottom);
+            if (navigationViewTop != null && navigationViewBottom != null) {
+                navigationViewTop.setNavigationItemSelectedListener(this);
+                navigationViewBottom.setNavigationItemSelectedListener(this);
+            }
+
+            drawer.findViewById(R.id.nav_view_top).bringToFront();
+            drawer.findViewById(R.id.nav_view_bottom).bringToFront();
+        }
+
+        ViewStub stub = (ViewStub) findViewById(R.id.content_base);
+        if (stub != null) {
+            stub.setLayoutResource(activityContent);
+            stub.inflate();
+        }
+
+        buildNewGoogleApiClient();
+    }
+
+    @Override
+    public void onBackPressed() {
+        if (drawer != null) {
+            if (drawer.isDrawerOpen(GravityCompat.START)) {
+                drawer.closeDrawer(GravityCompat.START);
+            } else {
+                super.onBackPressed();
+            }
+        } else {
+            super.onBackPressed();
+        }
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        // Inflate the menu; this adds items to the action bar if it is present.
+        getMenuInflater().inflate(R.menu.base, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        // Handle action bar item clicks here. The action bar will
+        // automatically handle clicks on the Home/Up button, so long
+        // as you specify a parent activity in AndroidManifest.xml.
+        int id = item.getItemId();
+
+        return id == R.id.action_settings || super.onOptionsItemSelected(item);
+
+    }
+
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
+        final Intent intent;
+
+        // Handle navigation view item clicks here.
+        int id = item.getItemId();
+
+        if (id == R.id.nav_car_information) {
+            intent = new Intent(this, CarInfoActivity.class);
+            startActivity(intent);
+        } else if (id == R.id.nav_search) {
+            intent = new Intent(this, StaticActivity.class);
+            startActivity(intent);
+        } else if (id == R.id.sign_out) {
+            Auth.GoogleSignInApi.signOut(googleApiClient).setResultCallback(
+                    new ResultCallback<Status>() {
+                        @Override
+                        public void onResult(Status status) {
+                            if (status.isSuccess()) {
+                                Intent intent = new Intent(BaseActivity.this, LoginActivity.class);
+                                startActivity(intent);
+                            }
+                        }
+                    });
+
+        }
+
+        drawer.closeDrawer(GravityCompat.START);
+        return true;
+    }
+
+    private void buildNewGoogleApiClient() {
```

Comment:
This is because Login Activity does not extend Base Activity (it doesn't have the app toolbar at the top). Pretty much all other activities extend Base Activity, so we should only need to build the Google API client twice.